src/hooks/useAudio/useAudio.test.ts

```diff
@@ -6,8 +6,13 @@ import useAudio from './useAudio';
 import { CreateAudioArgs } from './useAudio.types';
 
 describe('useAudio', () => {
+  beforeAll(() => {
+    window.HTMLMediaElement.prototype.load = jest.fn();
```

Comment:
This is a better approach. When I was writing the tests I was looking for the proper way to test it.